Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-# create a work dir and copy local files in
 WORKDIR /go/src/github.com/openshift/console-operator
 COPY . .
-RUN make build
+RUN ADDITIONAL_GOTAGS="ocp" make build WHAT="cmd/console"
```

Comment:
`ocp` by default, `okd` if this configmap exists (which the operator will merge):
https://github.com/openshift/origin-branding/pull/4/files#diff-26921f007ed806c079e9bb8160490990R12

just for review confirmation.